app/views/tools/index.html.haml

```diff
@@ -0,0 +1,26 @@
+.grid-row
+  .column-two-thirds
+    %h1.heading-xlarge
+      Tools
+    
+    %p 
+      These tools will help you complete your submission.
+    
```

Comment:
there's some rogue trailing whitespace in this file. worth looking into configuring your editor to catch and remove these automatically for you to avoid any more sneaking into the repo